mail-function/src/.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "env": {
+    "browser": true,
```

Comment:
```suggestion
```
You run your application with nodejs and not in the browser, so you don't need the browser env

Reply:
yes thats weird, when configuting eslint I selected node .. but thanks